server/accounts_test.go

```diff
@@ -3380,3 +3380,39 @@ func TestAccountLimitsServerConfig(t *testing.T) {
 	_, err = nats.Connect(s.ClientURL(), nats.UserInfo("derek", "foo"))
 	require_Error(t, err)
 }
+
+func TestAccountUserSubPermsWithQueueGroups(t *testing.T) {
+	cf := createConfFile(t, []byte(`
+	listen: 127.0.0.1:-1
+
+	authorization {
+	users = [
+		{ user: derek, password: "pass" }
+		{ user: user, password: "pass",
+			permissions: {
+				publish: "foo.restricted"
+				subscribe: { allow: "foo.>", deny: "foo.restricted" }
+				allow_responses: { max: 1, ttl: 0s }
+			}
+		}
+	]}
+    `))
+	defer removeFile(t, cf)
+
+	s, _ := RunServerWithConfig(cf)
+	defer s.Shutdown()
+
+	nc, err := nats.Connect(s.ClientURL(), nats.UserInfo("user", "pass"))
+	require_NoError(t, err)
+
+	// Now do same with qsub.
```

Comment:
Copy-paste from another test I presume, since we only do queue sub in this test.

Reply:
Yes, I had both to start but the trick was it had to be solo to present the issue. Will clean up test and comments.